server/router.js

```diff
@@ -1,17 +1,22 @@
 const express = require('express');
 const router = express.Router();
 
-router
-  .route('/')
-  .get((req, res) => {
-    console.log('got a get request!');
-    res.status(200).send('good job getting to this handler')
-  })
+// individual routes
+const products = require('./routes/products.js');
+const reviews = require('./routes/revews.js');
+const qa = require('./routes/qa.js');
+const cart = require('./routes/cart.js');
 
-// router
-//   .route('/')
+// edge case router for testing empty endpoint
+router.get('/', (req, res) => {
+  console.log('got a get request!');
+  res.status(200).send('good job getting to this handler, but you need to specify an endpoint for data.')
+})
```

Comment:
This entire 'edge case' actually doesn't need to be here, because the default directory should actually just serve us the static file